src/main/resources/templates/browse.html

```diff
@@ -39,20 +40,20 @@
             </div>
         </div>
     </nav>
-
+<!-- Browse movies or series -->
     <h1>Browse</h1>
     <div id="dashDiv"></div>
 
     <div class="pageBtnDiv">
-        <button id="prevPageBtn" class="btn btn-info pageBtn" style="visibility:hidden">< Previous Page</button>
-        <button id="nextPageBtn" class="btn btn-info pageBtn" style="visibility:hidden">Next Page ></button>
+        <button id="prevPageBtn" class="btn btn-info pageBtn hidden">< Previous Page</button>
+        <button id="nextPageBtn" class="btn btn-info pageBtn hidden">Next Page ></button>
```

Comment:
Ok, I see... adding hidden as a class.  Generally, yes, good idea to move styling away from inline, and into css.